doc/tor.1.txt

```diff
@@ -3369,260 +3319,260 @@ Tor catches the following signals:
-    how to set their Stable flags.
-
-__DataDirectory__**/stats/dirreq-stats**::
-    Only used by directory caches and authorities. This file is used to
-    collect directory request statistics.
-
-__DataDirectory__**/stats/entry-stats**::
-    Only used by servers. This file is used to collect incoming connection
-    statistics by Tor entry nodes.
-
-__DataDirectory__**/stats/bridge-stats**::
-    Only used by servers. This file is used to collect incoming connection
-    statistics by Tor bridges.
-
-__DataDirectory__**/stats/exit-stats**::
-    Only used by servers. This file is used to collect outgoing connection
-    statistics by Tor exit routers.
-
-__DataDirectory__**/stats/buffer-stats**::
-    Only used by servers. This file is used to collect buffer usage
-    history.
-
-__DataDirectory__**/stats/conn-stats**::
-    Only used by servers. This file is used to collect approximate connection
-    history (number of active connections over time).
-
-__DataDirectory__**/stats/hidserv-stats**::
-    Only used by servers. This file is used to collect approximate counts
-    of what fraction of the traffic is hidden service rendezvous traffic, and
-    approximately how many hidden services the relay has seen.
-
-__DataDirectory__**/networkstatus-bridges**::
-    Only used by authoritative bridge directories. Contains information
-    about bridges that have self-reported themselves to the bridge
-    authority.
-
-__DataDirectory__**/approved-routers**::
-    Authorities only. This file is used to configure which relays are
-    known to be valid, invalid, and so forth.
-
-__HiddenServiceDirectory__**/hostname**::
-    The <base32-encoded-fingerprint>.onion domain name for this hidden service.
-    If the hidden service is restricted to authorized clients only, this file
-    also contains authorization data for all clients.
- +
-    Note that clients will ignore any extra subdomains prepended to a hidden
-    service hostname. So if you have "xyz.onion" as your hostname, you
-    can tell clients to connect to "www.xyz.onion" or "irc.xyz.onion"
-    for virtual-hosting purposes.
-
-__HiddenServiceDirectory__**/private_key**::
-    The private key for this hidden service.
-
-__HiddenServiceDirectory__**/client_keys**::
-    Authorization data for a hidden service that is only accessible by
-    authorized clients.
-
-__HiddenServiceDirectory__**/onion_service_non_anonymous**::
-    This file is present if a hidden service key was created in
-    **HiddenServiceNonAnonymousMode**.
-
-SEE ALSO
---------
-**torsocks**(1), **torify**(1) +
-
-**https://www.torproject.org/**
-
-**torspec: https://spec.torproject.org **
-
-BUGS
-----
-
-Plenty, probably. Tor is still in development. Please report them at https://trac.torproject.org/.
+=== Files
+This section contains information about Tor's configuration files.
+
+[%header,cols=2*]
+|===
+|File Location
+|Description
+
+|`**@CONFDIR@/torrc**`
+|Contains _option-value_ pairs.
```

Comment:
Is there a reason we shouldn't say this is the configuration file?

Maybe we should say that this is the default location.